draft-trammell-taps-post-sockets.md

```diff
@@ -164,8 +164,8 @@ Post replaces the traditional SOCK_STREAM abstraction with an Message
 abstraction, which can be seen as a generalization of the Stream Control
 Transmission Protocol's {{RFC4960}} SOCK_SEQPACKET service. Messages are sent
 and received on Carriers, which logically group Messages for transmission and
-reception. For backward compatibility, these Carriers can also be opened as
-Streams, presenting a file-like interface to the network as with SOCK_STREAM.
+reception. For backward compatibility, byte stream protocols are represented
+as a Carrier with a single long message in each direction.
```

Comment:
Yes, I should wordsmith this. It's not necessarily bounded on carrier lifetime—if the carrier is considered to be bidirectional, then its lifetime would be lifetime of both directions. However, one message can be complete for half-closed TCP stream states, without the other being complete.